Maple2.Server.World/WorldServer.cs

```diff
@@ -1,28 +1,48 @@
-﻿using System;
+﻿using Maple2.Database.Extensions;
 using Maple2.Database.Storage;
+using Maple2.Model.Metadata;
+using Maple2.Server.World.Containers;
 using Maple2.Tools.Scheduler;
-using WorldClient = Maple2.Server.World.Service.World.WorldClient;
+using Serilog;
+using ChannelClient = Maple2.Server.Channel.Service.Channel.ChannelClient;
 
 namespace Maple2.Server.World;
 
 public class WorldServer {
     private readonly GameStorage gameStorage;
+    private readonly ChannelClientLookup channelClients;
+    private readonly ServerTableMetadataStorage serverTableMetadata;
+    private readonly GlobalPortalLookup globalPortalLookup;
+    private readonly Thread thread;
     private readonly EventQueue scheduler;
 
-    #region Autofac Autowired
-    // ReSharper disable MemberCanBePrivate.Global
-    public required WorldClient World { get; init; }
-    // ReSharper restore All
-    #endregion
+    private readonly ILogger logger = Log.ForContext<WorldServer>();
+
 
-    public WorldServer(GameStorage gameStorage) {
+    public WorldServer(GameStorage gameStorage, ChannelClientLookup channelClients, ServerTableMetadataStorage serverTableMetadata, GlobalPortalLookup globalPortalLookup) {
         this.gameStorage = gameStorage;
+        this.channelClients = channelClients;
+        this.serverTableMetadata = serverTableMetadata;
+        this.globalPortalLookup = globalPortalLookup;
         scheduler = new EventQueue();
+        scheduler.Start();
 
-        SetDailyReset();
+        StartDailyReset();
+        StartWorldEvents();
+        thread = new Thread(Loop);
+        thread.Start();
     }
 
-    public void SetDailyReset() {
+    private void Loop() {
+        while (true) {
+            scheduler.InvokeAll();
+            Thread.Sleep(15);
```

Comment:
is it needed that frequently?

Reply:
I guess not. What do you think would be best?

Reply:
1m should do it?